docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -90,6 +90,28 @@ following aggregate functions:
 * :func:`var_pop`
 * :func:`var_samp`
 
+Table property ``data_compression``
+-----------------------------------
+
+You can specify the data compression policy for SQL Server tables with the ``data_compression``
+Valid policies are ``NONE``, ``ROW`` or ``PAGE``.
+
+See `Data Compression
```

Comment:
I would get rid of this sentence and instead embed the link in the sentence above

You can specify the `data compression policy for SQL Server tables <url>`_ ....